conf/environments.js

```diff
@@ -26,6 +26,16 @@ module.exports = {
             globalReturn: true
         }
     },
+    commonjs: {
+        globals: {
+            module: false,
+            require: false,
+            exports: false
+        },
+        ecmaFeatures: {
+            globalReturn: true
```

Comment:
It does not in and of itself imply function scope? Solving the issue of `var Window = require('Window');` (shadowing a global variable rather than overwriting it)..